API.md

```diff
@@ -0,0 +1,50 @@
+# MobyDig API
+
+## `class MobyDig`
+Uses: class Spi2Parallel
+
+Create an object of type `MobyDig` to communicate with a particular [MobyDig 7-segment indicator modules](https://my.amperka.com/modules/mobydig).
+
+### `MobyDig(uint8_t pinCS, uint8_t devicesCount = 1)`
+
+Constructs a new object of data bus Hardware SPI with arguments:
+
+- `pinCS`: pin Chip Select used for controls the shift-register latch. Any GPIO-pin can be used.
+- `devicesCount`: quantity of `MobyDig` modules connected in one chain: logical output pin headers of one module to the inputs of another. If the `devicesCount` argument is omitted, it used only one module `MobyDig`.
```

Comment:
`devicesCount`: quantity of the `MobyDig` modules connected in a chain: logical output pin of one module is connected to the input of another. If the `devicesCount` argument is omitted, it used only one module `MobyDig`.

Reply:
done